lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/aten_ltc_ts_type.cpp

```diff
@@ -249,6 +252,159 @@ at::Tensor LazyNativeFunctions::_copy_from_and_resize(const at::Tensor& self,
   }
   return dst;
 }
+namespace ir {
+namespace ops {
+
+class ToCopy : public torch::lazy::TsNode {
```

Comment:
TODO(whc) - this should be moved to another file, obviously..

I actually first implement 'ToCopy' via codegen before realizing that I need to special-case the to(device) behavior, but it was still useful to copy-paste the codegenerated IR class, and use it for the to(dtype) cases.